SOURCE/FD_Jacobian.c

```diff
@@ -67,38 +67,27 @@ void PhaseRheologyLoop( int centroid, double sign, double denom, double Exx, dou
         }
         
         if ( model->eta_avg == 0) { // ARITHMETIC AVERAGE
```

Comment:
I've just noticed, that every action in the cycle could be placed under `if ( cond == 1 ) {`

Reply:
yes, this is part of the reformatting :D